rust/main/chains/hyperlane-sealevel/src/mailbox.rs

```diff
@@ -685,18 +685,18 @@ impl SealevelMailboxIndexer {
         &self.mailbox.rpc()
     }
 
-    async fn get_dispatched_message_with_nonce(
+    async fn get_dispatched_message_with_sequence(
```

Comment:
I don't feel strongly but fwiw I'm comfortable with referring to sequences as nonces for dispatched messages as they're the same. Calling this out mostly because in some other places we keep the nonce language (e.g. `retrieve_dispatched_message_by_nonce` that's in this PR), and consistency is nice. But agree that `get_delivered_message_with_sequence` is a better name

Reply:
I keep nonce for consistency.